services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -46,26 +46,24 @@ export async function main(event) {
     );
   }
 
-  const [getApplicationError, application] = await to(
-    vivaAdapter.application.get(user.personalNumber)
-  );
+  const [getApplicationError, vivaPerson] = await to(vivaAdapter.person.get(user.personalNumber));
   if (getApplicationError) {
     return console.error('(Viva-ms) Viva Get Application Request', getApplicationError);
   }
 
-  if (!application || !application.period) {
+  if (!vivaPerson.application || !vivaPerson.application.period) {
     return console.error('(Viva-ms) Viva Application Period not present in response, aborting');
   }
 
-  if (!application || !application.workflowid) {
+  if (!vivaPerson.application || !vivaPerson.application.workflowid) {
     return console.error(
-      `(Viva-ms) Viva Application WorkflowId ${application.workflowid} not present in response, aborting`
+      `(Viva-ms) Viva Application WorkflowId ${vivaPerson.application.workflowid} not present in response, aborting`
```

Comment:
Add `?` to `application` for the error message, otherwise this will fail in case it's invalid.